ntia-algorithm.sigmf-ext.md

```diff
@@ -18,6 +18,7 @@ A `DigitalFilter` object is used to describe a discrete-time linear time-invaria
 
 | name                       | required | type                                         | unit | description                                                                                                                                                       |
 |----------------------------|----------|----------------------------------------------|------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `type`                     | true     | string                                       | N/A  | MUST be set to `"DigitalFilter"`. Used to identify the object type when parsing.                                                                                  |
 | `id`                       | true     | string                                       | N/A  | Unique ID of the filter                                                                                                                                           |
 | `filter_type`              | true     | [enum](./ntia-core.sigmf-ext.md#0-datatypes) | N/A  | Type of the digital filter, given by the [FilterType](#011-the-filtertype-enum) enum                                                                              |
```

Comment:
Shouldn't the type of filter_type be FilterType and link to the FilterType Enum?

Reply:
Yeah, I think that makes sense. Fixed in f1edf5f454376842ffd5778b462601f402d0bc4f